agimus_controller/main/main_hpp_panda_mpc.py

```diff
@@ -1,15 +1,24 @@
 import time
+import os
+import example_robot_data
 from agimus_controller.hpp_interface import HppInterface
 from agimus_controller.mpc import MPC
 from agimus_controller.utils.plots import MPCPlots
+from agimus_controller.utils.build_models import get_robot_model, get_collision_model
 
 from agimus_controller.utils.wrapper_panda import PandaWrapper
 from agimus_controller.ocps.ocp_croco_hpp import OCPCrocoHPP
 
 if __name__ == "__main__":
     pandawrapper = PandaWrapper(auto_col=True)
+    current_dir_path = os.path.dirname(os.path.abspath(__file__))
+    urdf_path = os.path.join(current_dir_path, "../../urdf/robot.urdf")
```

Comment:
I let it pass this one time but we really need to start working with robot_description packages!!!!!